SPECS/sm.spec

```diff
@@ -154,6 +154,21 @@ if [ $1 -gt 1 ]; then
     multipathd reconfigure
 fi
 
+if [ $1 -gt 1 ]; then
+    # XCP-ng: we temporarily removed sr_health_check.timer,
+    # so we need to disable and stop it if it is present on the system
+    TIMER_LINK=/etc/systemd/system/timers.target.wants/sr_health_check.timer
+    if [ -e "$TIMER_LINK" ]; then
+        systemctl --no-reload disable sr_health_check.timer 2>&1 || :
+        systemctl stop sr_health_check.timer 2>&1 || :
+    elif [ -L "$TIMER_LINK" ]; then
+        # Remove dangling symlink left by previous build that removed
+        # the timer without disabling it first.
+        rm -f "$TIMER_LINK"
```

Comment:
If the goal is to remove a symlink, `unlink` is more adapted.

Reply:
As discussed, it's not obvious, and as I already merged the other PR (I've done it in the wrong order), I'll merge this one as is too.